css/homepage.scss

```diff
@@ -31,6 +31,67 @@
  }
 }
 
+#description {
+  margin-top: 5rem;
+  margin-bottom: 5rem;
+  .icon {
+    line-height: 1;
+    vertical-align: middle;
+    font-size: 20em;
+    color: #400727;
+  }
+  p {
+    line-height: 1.5em;
+    color: #6f7c82
+  }
+  ul {
+    display: flex;
+    font-size: 1.1em;
+    padding-top: 1em;
+    margin: 0;
+    padding: 0;
+    list-style: none;
+    li {
+      position: relative;
+      display: list-item;
+      &:first-child {
+        margin-right: 1.8em;
+        &:after {
+          content: "";
+          position: absolute;
+          right: -17px;
+          top: calc(50% - 2px);
+          width: 6px;
+          height: 6px;
+          background: #dde5eb;
+          box-shadow: inset 0 1px 1px rgba(127,153,176,.3);
+          border-radius: 3px;
+        }
+      }
+      a {
+        &:hover:before {
+          transform:none;
+          opacity:1;
+          transition-duration:.4s
+        }
+        &:before {
+          content: "";
+          position: absolute;
+          width: 100%;
+          height: 2px;
+          bottom: -4px;
+          background: #e5ebf0;
+          transform: translateY(7px);
```

Comment:
This all looks very fragile. Have you checked rendering when resizing the font of your browser?